recipe/meta.yaml

```
@@ -51,7 +51,7 @@ about:
     - cugraph/LICENSE
     - rmm/LICENSE
     - cugraph/cpp/build/gunrock/src/gunrock_ext/LICENSE.TXT
-    - cugraph/cpp/build/raft/src/raft/LICENSE
+    - cugraph/cpp/build/_deps/raft-src/LICENSE
     - cugraph/cpp/build/_deps/thrust-src/LICENSE
     - cugraph/cpp/build/_deps/cuco-src/LICENSE
     - cugraph/cpp/build/cuhornet/src/cuhornet/LICENSE
```

Looks like we need to fix this line too...
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=307538&view=logs&j=4f922444-fdfe-5dcf-b824-02f86439ef14&t=b2a8456a-fb11-5506-ca32-5ccd32538dc0&l=1032
```suggestion
    - cugraph/cpp/build/_deps/cuhornet-src/LICENSE
```

---

Yes, sorry about that. I was just typing that up here after I saw the failure.  I think that's the last one :)

---

No worries, Rick! There're so many deps and it's hard to get them right in one shot...Let's see how it goes!